openfisca_core/commons/typing.py

```diff
@@ -2,11 +2,9 @@
 
 from nptyping import NDArray as Array
 
-
-_T = TypeVar("_T", float, str)
-""":obj:`.TypeVar`: A wildcard type used by other types."""
-
-ArrayLike = Union[Union[Array[float], Array[str]], Sequence[_T]]
+_Types = TypeVar("_Types", float, int, str, bool)
+_Array = Union[Array[float], Array[int], Array[str], Array[bool]]
```

Comment:
Shoudn't we use alphabetical order ?
Or does the order is defined for testing purposes, the more likely being in front of the list ?

Reply:
Alphabetical makes sense :)